filip/utils/validators.py

```diff
@@ -80,3 +83,65 @@ def validate_escape_character_free(value: Any) -> Any:
                 raise ValueError(f"The value {value} contains "
                                  f"the forbidden char '")
     return values
+
+
+def match_regex(value: str, pattern: str):
+    regex = re.compile(pattern)
+    if not regex.match(value):
+        raise PydanticCustomError(
+            'string_pattern_mismatch',
+            "String should match pattern '{pattern}'",
+            {'pattern': pattern},
+        )
+    return value
+
+
+def ignore_none_input(func):
+    def wrapper(arg):
+        if arg is None:
+            return arg
+        return func(arg)
+    return wrapper
+
+
+def validate_fiware_standard_regex(vale: str):
+    from filip.models.base import FiwareRegex
```

Comment:
It is not necessary. But otherwise, I am facing the problem of nested import... so right now, I will move the definition of `FiwareRegex` to `validators.py`